services/user.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.auth import get_user_model
+from django.shortcuts import get_object_or_404
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
```

Comment:
The `create_user` function should validate the uniqueness of the username and email before attempting to create the user. Consider checking for existing users with the same username or email to prevent integrity errors.